src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.State.cs

```diff
@@ -226,6 +235,11 @@ private TExpressionSyntax GetExpressionUnderSpan(SyntaxTree tree, TextSpan textS
                 }
 
                 return commonExpression;
+
+                bool isWhitespaceOrEndOfLineTrivia(SyntaxTrivia trivia)
+                {
+                    return syntaxFacts.IsWhitespaceTrivia(trivia) || syntaxFacts.IsEndOfLineTrivia(trivia);
+                }
```

Comment:
i'm guessing this is in line 50 places in the IDE by now ;-)

Reply:
actually, shockingly, i can't find anything like this.  nit: consider adding this to ISyntaxFactServiceExtensions